src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: $card-width;
+  height: $card-height;
+  border-radius: $border-radius;
+  border: 1px solid $border-color;
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__product-details {
+    width: $card-content-width-adjustment;
+    margin-inline: $padding-inline;
+    margin-bottom: $margin-bottom;
+
+    &-title {
+      height: 36px;
+      font-size: $font-size-medium;
+      font-weight: $font-weight-medium;
+      line-height: 18px;
+      text-align: left;
+      color: $secondary-color;
+      margin-top: 0;
+      margin-bottom: 4px;
+    }
+
+    &-description {
+      margin: 0;
+      font-size: $font-size-small;
+      font-weight: $font-weight-regular;
+      line-height: 14px;
```

Comment:
As with the `line-height` in the `&-title`, it should be at least 1.5 times the font-size. Please revise the `line-height` value to improve text readability.